src/components/Dashboard/Form/Button/Button.js

```diff
@@ -2,17 +2,17 @@ import React from "react";
 import styles from "./Button.scss";
 import PropTypes from "prop-types";
 
-const Button = props => (
-  <div>
+const Button = props => {
+  return (
```

Comment:
We don't need this return. Convention like before (just without wrapping div) was perfectly fine

Reply:
done